articles/search/search-get-started-vector.md

```diff
@@ -286,18 +291,76 @@ api-key: {{admin-api-key}}
             -0.00086512347
         ],
         "fields": "contentVector",
+        "select": "title, content, category"
         "k": 10
     },
-    "select": "title, category",
     "filter": "category eq 'Databases'"
 }
 ```
 
+
+### Cross-field vector search
+Cross-field vector search allows you to send a single query across multiple vector fields in your vector index. For this example, I want to calculate the similarity across both `titleVector` and `contentVector`:
+
+```http
+POST https://{{search-service-name}}.search.windows.net/indexes/{{index-name}}/docs/search?api-version={{api-version}}
+Content-Type: application/json
+api-key: {{admin-api-key}}
+{
+    "vector": {
+        "value": [
+            -0.009154141,
+            0.018708462,
+            . . . 
+            -0.02178128,
+            -0.00086512347
+        ],
+        "fields": "titleVector, contentVector",
+        "k": 5
+    }
+}
+```
+
+### Multi-query vector search
+Multi-query vector search allows you to send a multiple queries across multiple vector fields in your vector index. For this example, I want to calculate the similarity across both `titleVector` and `contentVector` but will send in two different query embeddings respectively. This scenario is ideal for multi-modal use cases where you want to search over a `textVector` field and an `imageVector` field. You can also use this scenario if you have different embedding models with different dimensions in your search index. 
```

Comment:
might want to add a comment that the scores are RRF scores now, not similarity scores. Which would explain why the values are very low.